plugins/memcached.yaml

```diff
@@ -18,17 +17,32 @@ parameters:
     default: end
 
 # Set Defaults
-# {{$file_path := default "/var/log/memcached.log*" .file_path}}
 # {{$start_at := default "end" .start_at}}
 
 # Pipeline Template
 pipeline:
-  - id: memcached_input
-    type: file_input
+  - id: memcached_journald_input
+    type: journald_input
     include:
-      - {{ $file_path }}
+      - {{ .memcached_journald_log_path }}
     start_at: {{ $start_at }}
     labels:
       log_type: memcached
       plugin_id: {{ .id }}
-    output: {{.output}}
+
+  - id: memcached_filter
+    type: filter
+    expr: '$record._SYSTEMD_UNIT != "memcached.service"'
+
+  - type: severity_parser
```

Comment:
Huh. We should probably do this automatically with journald since the severities are all standardized. Similarly to how we just modified syslog. This is fine for now, but I'll make a stanza ticket